CMakeLists.txt

```diff
@@ -14,16 +14,16 @@ FetchContent_Declare(AndroidExtensions
     GIT_TAG 7d88a601fda9892791e7b4e994e375e049615688)
 FetchContent_Declare(arcana.cpp
     GIT_REPOSITORY https://github.com/microsoft/arcana.cpp.git
-    GIT_TAG f2757396e80bc4169f2ddb938ce25367a98ffdd0)
+    GIT_TAG 1a8a5d6e95413ed14b38a6ac9419048f9a9c8009)
 FetchContent_Declare(arcore-android-sdk
     GIT_REPOSITORY https://github.com/google-ar/arcore-android-sdk.git
     GIT_TAG 98cb803de5482fb2b36effe8be3b5a0d3b726976)
 FetchContent_Declare(base-n
     GIT_REPOSITORY https://github.com/azawadzki/base-n.git
     GIT_TAG 7573e77c0b9b0e8a5fb63d96dbde212c921993b4)
 FetchContent_Declare(bgfx.cmake
-    GIT_REPOSITORY https://github.com/BabylonJS/bgfx.cmake.git
-    GIT_TAG 345e3e28a9912983fc64d1dcb4da22a445afe3fd)
+    GIT_REPOSITORY https://github.com/okwasniewski/bgfx.cmake.git
```

Comment:
Just need this commit to be updated and it's good to go :)

Reply:
@CedricGuillemet we also need this one merged, right? https://github.com/BabylonJS/bgfx.cmake/pull/76

I've just pointed it at the last merged commit

Reply:
I've juste merged it.

Reply:
Great, I've changed the CMakeLists to point at upstream repo